ruby_review_week_four.rb

```diff
@@ -1,37 +1,135 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+can store data.
+
 
 # 2. What Ruby method can you use to make a loop from an array?
+ruby each loop
+
 
 # 3. In Ruby, what are booleans?
 
+a boolean refers to a value of either true or false, 
 # 4. How do classes relate to objects?
 
 # 5. What is an instance variable?
+class Person
+  def initialize(name)
+    @name = name
+  end
+end
+@name is a new type of variable, called an “instance variable”.
 
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
 
+
+
+
 # 7. What is handy about using gems in Ruby?
 
 # 8. Define a class called fish, and leave it empty.
+class Fish
+
+  def initialize(name, weight)
+    @name=name
+    @weight=weight
+  end
+end
+
+fash1=Fish.new("fash1","1kg");
+fash2=Fish.new("fash2","1.5kg");
+
+
+
 
 # 9. How would you use that class to make three fish objects?
 
+
+
+
 # 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.
+class Fish
+  attr_accessor :name :species
+  def initialize(name, species)
```

Comment:
this is great, make sure there's spaces between where you define your instance variables